src/desktop/analytics/main_layout.js

```diff
@@ -13,27 +14,46 @@ var properties = { path: location.pathname }
 // We exclude these routes from analytics.page calls because they're already
 // taken care of in Reaction.
 const excludedRoutes = [
-  "artist",
-  "artwork",
-  "campaign",
-  "collect",
-  "collection",
-  "collections",
-  "identity-verification",
-  "orders",
-  "search",
-  "viewing-room",
+  "/artist(.*)",
+  "/artwork(.*)",
+  "/auction-faq",
+  "/auction/:saleID/bid(2)?/:artworkID",
```

Comment:
This will now _only_ exclude this exact match, versus all things under the `auctions` route namespace. Makes the problem a little easier to resolve.